pom.xml

```diff
@@ -112,7 +112,7 @@
     <maven.version>3.8.6</maven.version>
     <exec-maven-plugin.version>1.6.0</exec-maven-plugin.version>
     <sbt.project.name>spark</sbt.project.name>
-    <slf4j.version>1.7.36</slf4j.version>
+    <slf4j.version>2.0.0</slf4j.version>
     <log4j.version>2.18.0</log4j.version>
```

Comment:
Log4j 2.18.0 does not support SLF4J 2.x. See https://github.com/apache/logging-log4j2/commits/968371c7a88ceb1c1ce15b70e3a5a08b49be7169/log4j-slf4j20-impl/pom.xml?browsing_rename_history=true&new_path=log4j-slf4j2-impl/pom.xml&original_branch=release-2.x
I think you will have to wait for LOG4J 2.19.

Slf4j 1.x uses static binding, whilst SLF4J 2.x uses ServiceLoader.

Reply:
Thanks @martin-g ~